spec/features/editing_file_attachments/delete_file_attachment_spec.rb

```diff
@@ -15,23 +25,45 @@ def given_there_is_an_edition_with_attachments
                       file_attachment_revisions: [@attachment_revision])
   end
 
+  def given_there_is_an_edition_with_featured_attachments
+    @attachment_revision = create(:file_attachment_revision, :on_asset_manager)
+
+    @edition = create(:edition,
+                      document_type: build(:document_type, :with_body, attachments: "featured"),
```

Comment:
We shouldn't need the `with_body` trait here, as it only applies for scenarios involving a body field.

Reply:
I followed the set up from `given_there_is_an_edition_with_attachments`. Does that mean that `:with_body` isn't needed there either?

Reply:
It is in that case: it's needed for scenarios involving a body field.

Reply:
I'm not sure I understand. Where is the body field being referenced in the "inline" tests?